src/views/Lobby/index.js

```diff
@@ -4,50 +4,68 @@ import propTypes from 'prop-types';
 import { connect } from 'react-redux';
 
 import {
-  Container, Content, Label, Button, Text, View, Separator
+  Container,
+  Content,
+  Label,
+  Button,
+  Text,
+  View,
+  ListItem,
+  Footer,
+  FooterTab,
```

Comment:
I don't think you want to use a dangling comma, I haven't seen it in other destructuring.

Reply:
`comma-dangle` is one of the linter rules that we set before.

Reply:
weird, I think that should force you to have dangling commas in all the initializations, not sure why it's only forcing you here, on the contrary, I like disallowing comma dangling because that avoids the thinking of when you can or cannot use it. https://eslint.org/docs/rules/comma-dangle


do the destructuring in all the other places like `views/Home/index.js:L31` need the dangling comma? it's weird to only see it enforced at the import level.